packages/enzyme/src/ReactWrapper.js

```diff
@@ -643,6 +643,7 @@ class ReactWrapper {
       const nodeHierarchy = [thisNode].concat(nodeParents(this, thisNode));
       renderer.simulateError(nodeHierarchy, rootNode, error);
 
+      this[ROOT].update();
```

Comment:
Even when I comment out this line, the tests still pass :-/

Reply:
Which react version are you using? I tested with 16.0 but will test with the others when I get back

Reply:
ah, duh, with 15 :-p let me try with 16

Reply:
@ljharb this change does fix the test in the example repo, but will try to rewrite the test here so that it fails properly on later versions

Reply:
Actually, running `npm run react:16.3 && npm run test:watch` fails for me when this line is commented

Reply:
If you rebase on master, you can run `npm run react 16.2` etc, and tests will also run against more react versions.

Reply:
Rebased, on react 16.{2-5} the added tests fail before the change and pass after.